docs/maps/vector-style.asciidoc

```diff
@@ -44,6 +44,11 @@ To enable data driven styling, click image:maps/images/gs_link_icon.png[] next t
 
 NOTE: The image:maps/images/gs_link_icon.png[] button is only available for vector features that contain numeric properties.
```

Comment:
I think we should avoid having back-to-back notes.  I suggest attaching this sentence to the previous paragraph and use this wording:

This button is only available for vector features that contain numeric properties.